api/BUILD

```diff
@@ -246,7 +249,6 @@ proto_library(
         "//envoy/service/discovery/v3:pkg",
         "//envoy/service/endpoint/v3:pkg",
         "//envoy/service/event_reporting/v3:pkg",
-        "//envoy/service/extension/v3:pkg",
```

Comment:
This shouldn't be removed.

Reply:
I found the culprit, fixing in #12367, meanwhile bring this line back.

Reply:
Done.